specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/preview/2020-07-01-preview/examples/AzureADMetricsCreate.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "rg1",
+    "azureADMetricsName": "ddb1",
+    "api-version": "2020-07-01-preview",
+    "azureADMetricsConfig": {
+      "location": "West US",
+      "tags": {}
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/rg1/providers/Microsoft.Aadiam/azureADMetrics/ddb1",
+        "name": "azureADMetricsName",
```

Comment:
name in both response bodies should be "ddb1" according to the parameters and the "id"